ci/releases/update-version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+########################
+#   Version Updater    #
```

Comment:
I'm curious what the intended use of this script is. I would expect the `jlpm version` command to be run manually which will update the stored version in `package.json` and create the git tag. Then on pushing of tags the deployment is automated from there.

Reply:
RAPIDS projects are automatically released when a merge to master occurs. This script is used to prepare for that release. After a merge into master, gpuCI determines what type of release (major, minor, patch), calls this script, makes the git commit/tag, and finally launches a build to publish the artifacts.

Reply:
Ah great thanks for the info!